sig-and-wg/wg-knative/knative-eventing-default-pubsub.md

```diff
@@ -0,0 +1,83 @@
+# Overview
+
+As an operator, I want to configure a default messaging middleware that my Kyma cluster should use for eventing.
+
+As a developer, I just want to configure a subscription for an event type from a source in Kyma and do not care about the underlying details.
+
+## Requirement
+
+* The pluggable Knative layer for the messaging middlewar (Knative ClusterChannelProvisioner) is available and deployable.
+* Operator will provision the pluggable layer into Kyma (Knative ClusterChannelProvisioner).
+  > Note: There will be some refernce examples that customers can follow. The provisioning/deployment will be an operator action as customes can choose a implementation specific to their needs.
+* Kyma is updated to use Knative version 0.4+.
+
+## The out-of-box Kyma installation
+
+The OOB Kyma installation will have NATS Streaming set as the default messaging middleware. The required Knative configurations will be set to use that.
+
+## When to specify the default?
+
+### Post Installation
+
+* Operator either creates a PubSub instance or use existing one.
+
+  >Note: For instance creation, she can use service catalog. Ths is not a must.
+
+* Operator deploys the cluster channel provisioner and injects the secrets.
+
+* Use [Knative semantics](https://github.com/knative/docs/blob/master/eventing/channels/default-channels.md#setting-the-default-channel-configuration) to specify the default ClusterChannelProvisioner.
```

Comment:
Can you describe how the developer will create a channel and will use the default provisioner? Can he still use other provisioners (not default)?

Reply:
There is another concept in-progress for this.